generator/src/main/java/com/scottlogic/deg/generator/config/detail/VisualiserLevel.java

```diff
@@ -0,0 +1,26 @@
+package com.scottlogic.deg.generator.config.detail;
+
+public enum VisualiserLevel  {
+    OFF {
+        @Override
+        public boolean sameOrHigherThan(VisualiserLevel level) {
+            return level == this;
+        }
+    },
+    STANDARD {
+        @Override
+        public boolean sameOrHigherThan(VisualiserLevel level) {
+            return OFF.sameOrHigherThan(level) || level == this;
+        }
+    },
+    DETAILED {
+        @Override
+        public boolean sameOrHigherThan(VisualiserLevel level) {
+            return true;
+        }
+    };
+
+    public abstract boolean sameOrHigherThan(VisualiserLevel level);
```

Comment:
this method seems more confusing than not having the method. it's only used in one place

Reply:
just doing one implementation and no overriding as its still small.  Will rename it so more obvious we want to know if level is same level of verbosity or more verbose than level passed in.  So its just:
    public boolean sameOrMoreVerboseThan(VisualiserLevel level) {
        return level == OFF || level == STANDARD && this != OFF || level == this;
    }